src/jquery.addressfield.js

```diff
@@ -179,6 +182,19 @@
   };
 
   /**
+  * Popualtes country dropdown with list of countries from provided json file if it is empty
+  */
+  $.fn.addressfield.initCountries = function(countryMap, selector){
+    $.each(countryMap, function(key, value) {
+       $(selector)
```

Comment:
You'll need to limit the selector to just the container that addressfield was initialized against:
```javascript
var $container = this;
$container.find(selector)
```

You'll see examples of this all over where we call `$fn.addressfield.method.call($container)` instead of `$fn.addressfield.method()`